density/static/css/style.css

```diff
@@ -300,6 +450,30 @@ img {
         padding: 10px;
     }
 
+    .searchTerm {
+        margin-bottom: 10px;
+    }
+
+    .searchButton {
+        right: 33px;
+    }
+
+    .searchContainer {
+        padding: 0 15px;
+    }
```

Comment:
This should be merged with the other `.searchContainer`

Reply:
Same as with the btnContainer styles, we want this padding to occur on .searchContainer only on mobile (it's within the media query).